packages/features/bookings/Booker/Booker.tsx

```diff
@@ -87,13 +87,15 @@ const BookerComponent = ({
   const date = dayjs(selectedDate).format("YYYY-MM-DD");
 
   const prefetchNextMonth =
-    (layout === BookerLayouts.WEEK_VIEW &&
+    !process.env.NEXT_PUBLIC_BOOKER_DISALLOW_PREFETCH_NEXT_MONTH &&
```

Comment:
Would rather call this ALLOW_PREFETCH_NEXT_MONTH so we don't have a double negative here but we need this to default to work as is. Without the double negative, it would be necessary for self-hosters upgrading to set this to `true` before upgrading, which is not ideal. 

As it stands now, the default will be false and will continue working as usual.